src/import/importd.c

```diff
@@ -1351,6 +1359,28 @@ static int manager_run(Manager *m) {
                         m);
 }
 
+static void parse_env(Manager *m) {
```

Comment:
Manager is an object we kinda want to use OOP-style programming for. Since this is C we do this by simply prefixing function names with `manager_`. Hence please rename this func to manager_parse_env() or so.

Reply:
Fixed.